.travis.yml

```
@@ -68,6 +68,14 @@ jobs:
       after_success: skip
       script: composer psalm
 
+    - stage: "JavaScript Unit Test"
```

```suggestion
    - stage: "Other tests"
```

Please use a stage defined in the stages block at top of this file